src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -270,7 +271,7 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
         if (!project.isDisposed) {
             SonarLintProjectNotifications.get(project).expireCurrentFindingNotificationIfNeeded()
         }
-        val file = tryFindFile(project, filePath.toString())
+        val file = tryFindFile(project, filePath.systemIndependentPath)
```

Comment:
Please see my comment on the other PR, I think we could try to call this method at a lower place